### DISCUSSION TITLE
Gap commands not working with I3 version 4.23

### DISCUSSION THREAD

**Discussion by ToddlerWit** [i3/i3#5839] [Q&A] [answered]
I'm using Zorin 17 with i3's Debian repository, I've tried running the gaps command normally and adding it to my .config. If I add gap commands to the .config it doesn't error it just does nothing. This is a fresh install. Everything I find talking about Debian i3 gaps is either pre-merge with i3-gaps, or irrelevant.

**Comment by okraits** [comment]
1. What exactly did you add to your config?
2. What is the exact error?
3. Did you look at the userguide?

**Reply by ToddlerWit** [reply, marked as answer]
Turns out, I'm an idiot. I did everything correctly but my terminal is the same color as my back ground and I set inner to be 5. I didn't even notice it was working perfectly.
